src/app-constants.js

```diff
@@ -16,6 +16,8 @@ const requiredEnvVars = [
   'SESSION_DURATION_HOURS',
   'SMTP_URL',
   'EMAIL_FROM',
+  'EMAIL_PREVIEW_ENABLED',
+  'EMAIL_RECIPIENT_DUMMY',
```

Comment:
Should these env vars be moved to "optional"?  i.e. should we block a dev from running the site if they're not set?